### ISSUE TITLE
Add: Transit --> EV Charging Stations 

### ISSUE THREAD

**Issue by Tatianaabor** [openooh/venue-taxonomy#114]
As EV charging station inventory is on the rise, and buyers are really interested in this type of screens, we should consider adding 'EV Charging Stations' as a child category under the existing Transit category.

Thoughts?

**Comment by jayshao** [comment]
I would think EV Charging would be parallel to or a sub-category of: Fueling Stations

(I can appreciate they may be deployed in smaller, less dedicated installations, and attached to other venues - e.g. in retail parking lots - but I would argue for the same relationship I think between Banks and ATMs... that the interaction is the same)

**Comment by jayshao** [comment]
I think this duplicates #105

**Comment by Tatianaabor** [comment]
Fueling Stations is a child category of Retail. I do not think people have to pay to get their EV charged, I believe it is a free service? If that's the case, we shouldn't classify it under Retail... Since EV chargers can be found in many different environments I think we should add it as a Parent category OR add it under the different environments where they could be found. Thoughts?

**Comment by jayshao** [comment]
In the US at least I think we have a strong mix of paid an "bundled with some manufacturer" providers.

From a pure logical perspective I don't necessarily disagree... though I am not sure the logical dissonance is enough I would be supportive of a chance to the the hierarchy for existing items... or splitting out a new category. I think the most important part is to have *a* category for it, and perhaps at some point if there is enough demand we would do a 2.x series that could make major changes to handling

**Comment by Tatianaabor** [comment]
Agreed! As long as the **EV charging station** classification is included in the existing venue taxonomy, that should serve the purpose.

**Comment by jayshao** [comment]
Closing as a duplicate of #105
